data/Filters/Roman/Roman-F062.ecsv

```diff
@@ -0,0 +1,85 @@
+# %ECSV 1.0
```

Comment:
can you not add these files, as they will be generated when running the code and save disk, git and download space. I'd recommend only adding explicitly files to git that you write and want to have added

Reply:
yes, that could work. Generally only add the specific files that you want